code/modules/games/cards.dm

```diff
@@ -0,0 +1,469 @@
+/datum/playingcard
+	var/name = "playing card"
+	var/card_icon = "card_back"
+	var/back_icon = "card_back"
+
+/obj/item/weapon/deck
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/playing_cards.dmi'
+	var/list/cards = list()
+	var/cooldown = 0 // to prevent spam shuffle
+	throw_speed = 3
+	throw_range = 10
+	throwforce = 0
+	force = 0
+
+/obj/item/weapon/deck/holder
+	name = "card box"
+	desc = "A small leather case to show how classy you are compared to everyone else."
+	icon_state = "card_holder"
+
+/obj/item/weapon/deck/cards
+	name = "deck of cards"
+	desc = "A simple deck of playing cards."
+	icon_state = "deck"
+
+/obj/item/weapon/deck/cards/New()
+	..()
+	var/datum/playingcard/P
+	for(var/suit in list("spades","clubs","diamonds","hearts"))
+
+		var/colour
+		if(suit == "spades" || suit == "clubs")
+			colour = "black_"
+		else
+			colour = "red_"
+
+		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "[colour]num"
+			P.back_icon = "card_back"
```

Comment:
Why do you explicitly set this here, but do not for the joker, when the default for the `back_icon` makes it so that it doesn't matter whether you assign it or not?